organizations.go

```diff
@@ -9,9 +15,28 @@ type Organization struct {
 	Roles       []string `json:"roles,omitempty"`
 }
 
-// OrganizationResponse represents the response from the Organization endpoint.
-type OrganizationResponse struct {
+// organizationResponse represents the response from the Organization endpoint.
+type organizationResponse struct {
 	Response
 	Result     []Organization `json:"result"`
 	ResultInfo ResultInfo     `json:"result_info"`
```

Comment:
Go lets you omit the second `ResultInfo` here but I'm OK either way having it or not.